docs/BreakingChanges/020 System.Activities is now APTCA.md

```diff
@@ -0,0 +1,22 @@
+## 20: System.Activities is now APTCA
```

Comment:
{
  "Id": "20",
  "Title": "System.Activities is now APTCA",
  "Details": "The assembly is marked with the AllowPartiallyTrustedCallersAttribute attribute.",
  "Suggestion": "Derived classes cannot be marked with the SecurityCriticalAttribute. Previously, derived types had to be marked with the SecurityCriticalAttribute. However, this change should have no real impact.",
  "Link": "https://msdn.microsoft.com/en-us/library/hh367887",
  "Markdown": null,
  "ApplicableApis": [
    "Not detectable via API analysis"
  ],
  "Related": null,
  "VersionBroken": "4.5",
  "VersionFixed": null,
  "IsBuildTime": false,
  "IsQuirked": false,
  "IsSourceAnalyzerAvailable": true,
  "BugLink": null,
  "Notes": null,
  "IsRetargeting": false,
  "ImpactScope": 3
}

**Parsed APIs:**
[ ] Not detectable via API analysis